### ISSUE TITLE
Skybox position should always be inherited from camera position

### ISSUE THREAD

**Issue by jcarpenter** [vr-components/vr-markup#8]
Moving the camera in the VR world should have no effect on the look of the skybox. Skyboxes should always appear to be at infinity.

**Comment by nickdesaulniers** [comment]
Is the skybox not anchored at the origin?

**Comment by jcarpenter** [comment]
I'm not sure I understand the question. We always drop Skybox at 0,0,0 (the default for all objects). The problem occurs when the user begins to walk through the scene. Although the issue is much less pronounced now that the default walking speed is roughly 1 meter / second, and the default sky sizes are approx 10,000 meters.

**Comment by sirkitree** [comment]
I agree that making it a child of the camera is not intuitive - but honestly I'm not sure having a 'camera' is intuitive to begin with (at least when your devs are coming from the web world).

Currently you don't have to define the camera with aframe though and it's defined with the a-sky component. If I walk forward for a long time I don't see any problems with distortion or anything and it seems that I could walk forward indefinitely.

So perhaps this is no longer a problem?

**Comment by jcarpenter** [comment]
The issue is currently obscured, but still present. The default WASD controls move the camera at 1m / second. The default `<a-sky>` radius is 5000 meters. If you add an image to the sky and walk around, you can perceive a subtle shift in perspective relative to the background. If users tweak the WASD controls to increase speed, or if they animate the camera via `<a-animation>`, this issue will inevitably rear it's head again. Increasing sky size will remain as a simple blunt-force solution, but that has performance impact (higher near/far spread to ensure sky is always within range of the camera `far` value).

Not a high priority, versus other outstanding items, but I'd love a better solution for this.

**Comment by ngokevin** [comment]
Could probably be solved with a component that defines the behavior of an entity following another entity.
